exercises/anagram/src/anagram.h

```diff
@@ -23,8 +23,8 @@ struct candidates {
 
 /**
  * @description - determines if any of the words in candidate are anagrams
- *                for word.  Word buffer and candidate structures may be modified.
+ *                for subject. Subject buffer and candidate structures may be modified.
```

Comment:
I think this might have been an error in the original. The subject/word buffer cannot be modified.